entity-framework/core/miscellaneous/cli/dbcontext-creation.md

```diff
@@ -49,8 +65,18 @@ namespace MyProject
 ```
 
 > [!NOTE]
-> The `args` parameter is currently unused. There is [an issue][2] tracking the ability to specify design-time arguments
+> The `args` parameter is currently unused. There is [an issue][7] tracking the ability to specify design-time arguments
 > from the tools.
 
-  [1]: https://docs.microsoft.com/aspnet/core/migration/1x-to-2x/#update-main-method-in-programcs
-  [2]: https://github.com/aspnet/EntityFrameworkCore/issues/8332
+A design-time factory can be especially useful if you need to configure the DbContext differently for design time than
+at run time, if the `DbContext` constructor take additional parameters are not registered in DI, if you are not using
+DI at all, or if for some reason you prefer not to have a `BuildWebHost` method in your ASP.NET Core application's  
+`Main` class.
+
+  [1]: xref:core/managing-schemas/migrations/index
+  [2]: xref:core/miscellaneous/configuring-dbcontext
+  [3]: https://docs.microsoft.com/aspnet/core/migration/1x-to-2x/#update-main-method-in-programcs
+  [4]: xref:core/miscellaneous/configuring-dbcontext#constructor-argument
+  [5]: xref:core/miscellaneous/configuring-dbcontext#using-dbcontext-with-dependency-injection
+  [6]: xref:core/miscellaneous/configuring-dbcontext#onconfiguring
+  [7]: https://github.com/aspnet/EntityFrameworkCore/issues/8332
```

Comment:
@bricelam I tried to keep the style used when you created this document. I think I succeeded, but feel free to point out anything I should change. BTW, as expected it was a real pain :smile: